minheap-internal.h

```diff
@@ -70,7 +70,7 @@ struct event* min_heap_top_(min_heap_t* s) { return s->n ? *s->p : 0; }
 
 int min_heap_push_(min_heap_t* s, struct event* e)
 {
-	if (min_heap_reserve_(s, s->n + 1))
+	if (s->n == UINT32_MAX || min_heap_reserve_(s, s->n + 1))
```

Comment:
> On 64 bit systems with a lot of memory it is possible to overflow the
counter s->n (unsigned int) which keeps track of the amount of elements
within the heap.

I'm suggesting to convert `s->n` to `size_t` instead

Reply:
Okay, will adjust that. I am discussing changes to libevent on OpenBSD's code base (1.x) as well, so it is nice to hear that it is acceptable to make this change for 2.x.

Reply:
If you are talking about converting to `size_t` it is definitely acceptable